src/collective/cover/layout.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from collective.cover.grid import BaseGrid  # noqa
+from zope import deprecation
+
+
+deprecation.deprecated(
```

Comment:
there was a typo here.. instead of ``SchemaNameEncoder`` should be ``BaseGrid``, fixed.